Makefile

```diff
@@ -111,7 +113,8 @@ unit-tests: manifests generate fmt vet setup-envtest ## Run unit tests.
 
 .PHONY: setup-envtest integration-tests
 integration-tests: manifests generate fmt vet setup-envtest ## Run integration tests.
-	ACK_GINKGO_DEPRECATIONS=2.12.0 KUBEBUILDER_ASSETS="$(KUBEBUILDER_ASSETS)" go test ./pkg/... ./controllers/... -ginkgo.v -ginkgo.progress -test.v -coverprofile cover.out
+	ACK_GINKGO_DEPRECATIONS=2.12.0 KUBEBUILDER_ASSETS="$(KUBEBUILDER_ASSETS)" go test ./pkg/... -ginkgo.v -ginkgo.progress -test.v -coverprofile cover.out
```

Comment:
calling `go test` twice is needed since we have different envtest configuration, otherwise the real cluster tests will not show log in the console for some reason. However, we are planning to rewrite the webhooks tests as unit tests so this should go away soonish.

Reply:
@flavio what about adding this to #579 instead?

Reply:
even better!